src/ioguild.cpp

```diff
@@ -30,6 +30,8 @@ Guild* IOGuild::loadGuild(uint32_t guildId)
 	query << "SELECT `name` FROM `guilds` WHERE `id` = " << guildId;
```

Comment:
Does this even work? you set houseId and OwnerId in line 33 and 34. But the query don't retrieve it from the SQL query?

This should solve it, it also takes @DSpeichert requested change into consideration:
```suggestion
	query << "SELECT `g`.`ownerid`, `g`.`name`, CASE WHEN `h`.`id` IS NULL THEN 0 ELSE `h`.`id` END AS `house_id` FROM `guilds` AS `g` LEFT JOIN `houses` AS `h` ON `g`.`id` = `h`.`guild_id` WHERE `g`.`id` = " << guildId << " LIMIT 1";
```

Beautified representation:
```sql
SELECT 
	`g`.`ownerid`, 
	`g`.`name`, 
	CASE WHEN `h`.`id` IS NULL 
		THEN 0 
		ELSE `h`.`id` 
	END AS `house_id` 

FROM `guilds` AS `g` 

LEFT JOIN `houses` AS `h` 
	ON `g`.`id` = `h`.`guild_id` 

WHERE `g`.`id` = " << guildId << " 
LIMIT 1
```
The case statement converts the output from "null" to 0 if no house_id was matched. Also renamed houses.id to house_id to avoid confusion down the line. 

Limit 1 is added to ensure that we only get 1 row back, could be a db bug where a guild has two houses, that would be problematic since its not accounted for.

Reply:
Well, why not allow guilds to have more than one house? There's nothing stopping players from having multiple houses - nothing other than Lua.

Reply:
Trying not to overcomplicate things and get a functional version merged asap. To support more guildhalls alot of changes has to be done in this pull request(right?), not just my suggested query change.

Reply:
From a pure SQL standpoint, there is nothing preventing that in future. We don't need it right now.